frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.styled.tsx

```diff
@@ -14,12 +14,15 @@ export const ColumnNameContainer = styled.div`
   display: flex;
   align-items: center;
   flex-shrink: 0;
+  width: 100%;
```

Comment:
The inner border/separator is not full-height: [this PR](https://github.com/metabase/metabase/assets/6830683/788fd45a-c957-4b58-b24a-7e6ddaf630ab) vs [0.49.11](https://github.com/metabase/metabase/assets/6830683/7e361c36-b617-4d45-b38a-c5a728af7fcf)